packages/friends/yml/en/james-parnell.yml

```diff
@@ -36,6 +36,35 @@ documents:
           - type: pdf
           - type: epub
           - type: mobi
+          - type: audio
+        audio:
+          reader: Jason Henderson
+          parts:
+            - title: Part 1
+              external_id_hq: TODO
+              external_id_lq: TODO
+              filesize_hq: TODO
+              filesize_lq: TODO
+              seconds: TODO
```

Comment:
did these TODOs go in before i got your local msf site running again?  Are they easy to fix with the correct info?  If so, let's do that now, but if not, just leave it.

Reply:
Oh, I just realized, these `TODO`s are actually causing the build failure on CI.  So they would either need to be removed, or fixed after all.

Reply:
I killed the TODOS... I never got my local msf site running.